src/test/java/domain/position/RouteTest.java

```diff
@@ -0,0 +1,255 @@
+                            new Position(File.D, Rank.FIVE)
+                    ));
+        }
+
+        @Test
+        void UP_LEFT_방향으로_이동하는_경로를_반환한다() {
+            Position source = new Position(File.E, Rank.FOUR);
+            Position target = new Position(File.A, Rank.EIGHT);
+
+            Route route = Route.create(source, target);
+            assertThat(route.getRoute())
+                    .hasSize(3)
+                    .containsExactlyInAnyOrderElementsOf(List.of(
+                            new Position(File.D, Rank.FIVE),
+                            new Position(File.C, Rank.SIX),
+                            new Position(File.B, Rank.SEVEN)
+                    ));
+        }
+
+        @Test
+        void UP_UP_LEFT_방향으로_이동하는_경로를_반환한다() {
+            Position source = new Position(File.E, Rank.FOUR);
+            Position target = new Position(File.D, Rank.SIX);
+
+            Route route = Route.create(source, target);
+            assertThat(route.getRoute()).hasSize(0);
+        }
+
+        @Test
+        void UP_UP_RIGHT_방향으로_이동하는_경로를_반환한다() {
+            Position source = new Position(File.E, Rank.FOUR);
+            Position target = new Position(File.F, Rank.SIX);
+
+            Route route = Route.create(source, target);
+            assertThat(route.getRoute()).hasSize(0);
+        }
+
+        @Test
+        void RIGHT_RIGHT_UP_방향으로_이동하는_경로를_반환한다() {
+            Position source = new Position(File.E, Rank.FOUR);
+            Position target = new Position(File.G, Rank.FIVE);
+
+            Route route = Route.create(source, target);
+            assertThat(route.getRoute()).hasSize(0);
+        }
+
+        @Test
+        void RIGHT_RIGHT_DOWN_방향으로_이동하는_경로를_반환한다() {
+            Position source = new Position(File.E, Rank.FOUR);
+            Position target = new Position(File.G, Rank.THREE);
+
+            Route route = Route.create(source, target);
+            assertThat(route.getRoute()).hasSize(0);
+        }
+
+        @Test
+        void DOWN_DOWN_RIGHT_방향으로_이동하는_경로를_반환한다() {
+            Position source = new Position(File.E, Rank.FOUR);
+            Position target = new Position(File.F, Rank.TWO);
+
+            Route route = Route.create(source, target);
+            assertThat(route.getRoute()).hasSize(0);
+        }
+
+        @Test
+        void DOWN_DOWN_LEFT_방향으로_이동하는_경로를_반환한다() {
+            Position source = new Position(File.E, Rank.FOUR);
+            Position target = new Position(File.D, Rank.TWO);
+
+            Route route = Route.create(source, target);
+            assertThat(route.getRoute()).hasSize(0);
+        }
+
+        @Test
+        void LEFT_LEFT_DOWN_방향으로_이동하는_경로를_반환한다() {
+            Position source = new Position(File.E, Rank.FOUR);
+            Position target = new Position(File.C, Rank.THREE);
+
+            Route route = Route.create(source, target);
+            assertThat(route.getRoute()).hasSize(0);
```

Comment:
```suggestion
            assertThat(route.getRoute()).isEmpty();
```

assertJ 에서 제공하는 api 를 최대한 활용해보아요 :)